assemblies/assembly-release-notes.adoc

```diff
@@ -28,5 +28,11 @@ include::{enterprise-dir}/release-notes/rn-7.10-known-issues-ref.adoc[leveloffse
 
 include::{enterprise-dir}/release-notes/rn-7.10-fixed-issues-ref.adoc[leveloffset=+1]
 
+ifdef::PAM[]
+include::{enterprise-dir}/release-notes/rn-7.10.1-known-issues-ref.adoc[leveloffset=+1]
```

Comment:
[question] Is it correct that 7.10.1 known issues is PAM only?

Reply:
That's correct. This time known issues are applicable just to RHPAM.